lib/scholar/metrics/classification.ex

```diff
@@ -700,13 +872,15 @@ defmodule Scholar.Metrics.Classification do
 
     case average do
       :none ->
-        {precision, recall, per_class_fscore}
+        support = (y_true == Nx.iota({num_classes, 1})) |> Nx.sum(axes: [1])
+
+        {precision, recall, per_class_fscore, support}
 
       :micro ->
-        {precision, recall, per_class_fscore}
+        {precision, recall, per_class_fscore, Nx.Constants.nan()}
```

Comment:
Any idea on what to add here? The first try was to use `:none`, but it seems atoms can't be returned from `defn`.

Reply:
Using NaNs seems reasonable

Reply:
I would personally prefer to have something more clear that this value contains "nothing" useful. But with the current constraints I thought this was ok(ish). Maybe a new type/struct could be created for these cases?